srcpkgs/img2pdf/template

```diff
@@ -1,14 +1,20 @@
 # Template file for 'img2pdf'
 pkgname=img2pdf
-version=0.4.4
-revision=3
+version=0.5.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-pikepdf python3-setuptools"
 depends="colord python3-pdfrw python3-pikepdf python3-Pillow python3-tkinter"
+checkdepends="$depends exiftool ghostscript ImageMagick mupdf-tools netpbm
+ poppler python3-numpy python3-pytest python3-pytest-xdist python3-scipy"
 short_desc="Lossless conversion of raster images to PDF"
 maintainer="Philipp David <pd@3b.pm>"
 license="GPL-3.0-or-later"
 homepage="https://gitlab.mister-muffin.de/josch/img2pdf"
 distfiles="${PYPI_SITE}/i/img2pdf/img2pdf-${version}.tar.gz"
-checksum=8ec898a9646523fd3862b154f3f47cd52609c24cc3e2dc1fb5f0168f0cbe793c
-make_check=no # need to patch out some bitdepth checks
+checksum=ae6c19731bde2551356c178bf356ca118ac32a232c737a14b423f8039df3c24b
+
+if [ "$XBPS_TARGET_WORDSIZE" != "64" ]; then
+	# bit depth checks fail on non-64bit arches
```

Comment:
this probably indicates an actual issue with assuming e.g. 64-bit types, it would be better to find this and fix it

Reply:
Not an expert on this, but it's probably how ImageMagick calculates bit depth and is intended. I'll open an issue with img2pdf about tests failing on i686 (and probably other 32-bit arches).